src/pages/Search/SearchPageBottomTab.tsx

```diff
@@ -35,6 +38,8 @@ function SearchPageBottomTab() {
     const styles = useThemeStyles();
     const StyleUtils = useStyleUtils();
     const [selectionMode] = useOnyx(ONYXKEYS.MOBILE_SELECTION_MODE);
+    const [narrowSearchRouterActive, setNarrowSearchRouterActive] = useState(false);
```

Comment:
rename similarly like I suggested in my other comment.
`Active` -> either `Focused` or `Visible`